serial_gui.py

```diff
@@ -1,10 +1,26 @@
 import tkinter
+from tkinter.constants import END
+from typing import Text
```

Comment:
these imports arent used, they can be deleted

Reply:
removed